### PR TITLE
NIAD-827: solving queue issues on AWS

### PR DESCRIPTION
**Pull request by bartek-sarul**
## Description

Add logging to `MeshInboundQueueService` to verify queue name and activity

## Jira Ticket

https://gpitbjss.atlassian.net/browse/NIAD-827

## Checklist

These are items (excluding GitHub Checks) which should be confirmed before a branch is ready to merge.

- [ ] Acceptance Criteria met
- [ ] Commit messages are meaningful
- [ ] Manually tested
- [ ] Self-reviewed your code
- [ ] Code reviewed from two other developers
- [ ] If your pull request depends on any other, please link them in the description
- [ ] main branch is passing/green on CI
- [ ] Add/update any relevant documentation

### CODE REVIEW

**Review by andrewwa-kainos — DISMISSED** [review]

**Review by saileshp-kainos — DISMISSED** [review]

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ logging:
     uk.nhs.digital.nhsconnect.lab.results: ${LAB_RESULTS_LOGGING_LEVEL:INFO}
 
 management:
-  endpoints:
+  endpoints:MeshInboundQueueService
```
Comment (matthewdoreilly) [inline src/main/resources/application.yml:9]: Should be a space here, `endpoints: MeshInboundQueueService`

File: src/main/resources/logback.xml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <configuration>
 	<appender name="TEXT" class="ch.qos.logback.core.ConsoleAppender">
 		<encoder>
-			<pattern>${LAB_RESULTS_LOGGING_FORMAT:-%d{yyyy-MM-dd HH:mm:ss.SSS} | %-5level | %thread | %logger{36} | %X{CorrelationId} %msg%n}</pattern>
+			<pattern>${LAB_RESULTS_LOGGING_FORMAT:-%d{yyyy-MM-dd HH:mm:ss.SSS} | %-5level | %thread | %logger{36} | %X{CorrelationId} | %msg%n}</pattern>
```
Comment (matthewdoreilly) [inline src/main/resources/logback.xml:4]: What's the reason for this change?

Comment (bartek-sarul) [reply src/main/resources/logback.xml:4]: correlation id is a separate field so should be pipe separated

**Review by andrewwa-kainos — APPROVED** [review]
